packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -60,7 +85,24 @@ export const initializeNodeEnvironment: EnvironmentInitializer<
     child.stdout?.setEncoding('utf8');
     child.stderr?.setEncoding('utf8');
     child.stdout?.on('data', console.log); // eslint-disable-line no-console
-    child.stderr?.on('data', console.error); // eslint-disable-line no-console
+
+    let lastErrors: string[] = [];
+    let lastErrorDateStamp: number | undefined = undefined;
+
+    child.stderr?.on('data', (chunk) => {
+        console.error(chunk); // eslint-disable-line no-console
+
+        if (lastErrorDateStamp === undefined) {
+            lastErrorDateStamp = Date.now();
+        }
+
+        // collect all errors if difference between occurrences is less than some threshold
+        if (lastErrorDateStamp - Date.now() < 5 * 1000) {
```

Comment:
Never use `Date.now()` to measure time intervals. You could even get a negative interval.

Reply:
fixed this